book/controller.rst

```diff
@@ -438,7 +438,7 @@ If you want to redirect the user to another page, use the ``redirectToRoute()``
         return $this->redirectToRoute('homepage');
 
         // redirectToRoute is equivalent to using redirect() and generateUrl() together:
-        // return $this->redirect($this->generateUrl('homepage'), 301);
```

Comment:
By default, both `redirect()` and `redirectToRoute()` do a `302` not a `301`, which must be explicitely pass to the function.